README.md

```diff
@@ -10,6 +10,18 @@ This library aims to complement [@daml/react](`https://www.npmjs.com/package/@da
 to make it easier to write [React](https://reactjs.org/) applications on DABL specifically.
 To that end we provide for convenience.
 
+## Install
+
+```
+yarn add digital-asset/dabl-react
+```
```

Comment:
Do you need `git` here

Reply:
I believe so

Reply:
You meant that you *don't* need to prefix the url with "git" . I just tried that and it worked. Which is insane, does it search both npmjs and github? (What if there's a conflict?)

Reply:
Oh wait a second! npmjs is now a part of github. Now that makes sense.